collection_requirements.rst

```diff
@@ -49,7 +49,7 @@ Collections
 * MUST be published to `Ansible Galaxy <https://galaxy.ansible.com>`_.
 * SHOULD NOT contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB like, for example, package installers for testing purposes.
 * SHOULD NOT contain any unnecessary files like, for example, temporary files created by development tools.
-* MUST only contain objects that follow the :ref:`licensing rules<Licensing>`.
+* MUST only contain objects that follow the :ref:`licensing rules<Licensing>`_.
```

Comment:
is this a typo?

Reply:
as i know this format is used to link external pages but the tests fail without this

Reply:
okay then ;)

Reply:
I think it should be either `:ref:` with double backticks and no underscore, or no `:ref:`, and then double backticks with single or double underscore.

Reply:
everything should be fine with the original version according to https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

If i remove the underscore, it returns:
```
Run rstcheck --report-level error *.rst
collection_requirements.rst:52: (ERROR/3) Unknown interpreted text role "ref".
Error! Issues detected.
Error: Process completed with exit code 1.
```

Reply:
`:ref:` is Sphinx specific, so it won't work in standalone RST files that aren't processed by Sphinx. The latest version in this PR definitely works though, since it is generic RST.